src/PactNet.Native/Messaging/MessageMiddleware.cs

```diff
@@ -0,0 +1,91 @@
+#if NETSTANDARD2_0_OR_GREATER
+
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Http;
```

Comment:
I think this middleware should be moved to a separate package entirely, because it ties the Native package to ASP.Net Core. We should probably have some kind of `PactNet.AspNetCore` package so that people who were using things other than ASP.Net Core (such as classic ASP.Net or NancyFx) can still use Pact.

I think the fact it has to be wrapped in a .Net Standard 2.0+ guard probably reinforces that even more.

Reply:
Good remark. I'll see to add it. Later we could add any number of package for the versions of the middlewares.